### ISSUE TITLE
DTQ with ResNet50

### ISSUE THREAD

**Issue by hbellafkir** [thulab/DeepHash#40]
hey @bl0 ,
I'm training DTQ with Resnet50 as Backbone on coco. I followed the suggested parameters in #21 with --n-part=100 and --triplet-margin=10, and it seems like the loss after the the first epoch does not converge: 
![Screenshot from 2019-06-28 10-33-38](https://user-images.githubusercontent.com/49009668/60329230-3ed50900-9990-11e9-9eab-deafd59bed7e.png)
 
Is the triplet-margin in this case too small?

**Comment by hbellafkir** [comment]
I would appreciate if you could answer me. I changed also the step decay for the learning to 20.000, but I keep getting the same problem. after the first epoch the loss stays constant.

**Comment by bl0** [comment]
Sorry for the late reply. We actually did not have tried the Resnet50. I think it's overfitting, maybe you can try to increase n-part or weight decay.

**Comment by hbellafkir** [comment]
ok I will try it.
